File: api/src/customization/fard.constants.ts
export const FARD_CATEGORY_CODES = new Set([
  'fajr',
  'dhuhr',
  'asr',
  'maghrib',
  'isha',
]);

export function isFardCategoryCode(code: string): boolean {
  return FARD_CATEGORY_CODES.has(code);
}

File: app/lib/features/customization/domain/fard_constants.dart
/// Default category codes that cannot be hidden or renamed (Phase 7 §3.2).
const Set<String> fardCategoryCodes = {
  'fajr',
  'dhuhr',
  'asr',
  'maghrib',
  'isha',
};

bool isFardCategoryCode(String code) => fardCategoryCodes.contains(code);
